tests/query/allocation_policies/test_concurrent_rate_limit_policy.py

```diff
@@ -144,3 +144,11 @@ def test_tenant_selection(policy):
     )
     with pytest.raises(AllocationPolicyViolation):
         policy._get_tenant_key_and_value({})
+
+
+def test_pass_through(policy: ConcurrentRateLimitAllocationPolicy) -> None:
```

Comment:
I don't see an assertion in this test. Is that intended?

Reply:
yes it is intended. If this was not a passthrough referrer, the request would be rejected

Reply:
Looking at the code it seems like to reject a request an exception is raised. If that is true, you could potentially explicitly check that exception is not raised in the test. Just makes things a bit more clearer. Something along the lines of the following. But this is not blocking so will approve the change
```
try:
   policy.get_quota_allowance()
except Exception:
   pytest.fail("Reason")
```